conformance-suites/1.0.2/conformance/resources/tex-image-and-sub-image-2d-with-canvas.js

```diff
@@ -83,6 +83,20 @@ function generateTest(pixelFormat, pixelType, prologue) {
       setCanvasToRedGreen(ctx);
     }
 
+    function checkSourceCanvasImageData(imageDataBefore, imageDataAfter) {
```

Comment:
The working group only updates the earlier conformance suite snapshots in order to fix bugs, and this is adding new tests. Could you please undo this change? Thanks.